src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -128,7 +191,7 @@ protected void addOriginEntriesForCorporateCardLines(
      * @param offsetGenerator A BiFunction that may create an offset entry using the generated regular entry and the detailLines List.
      */
     protected void addOriginEntriesForLines(
-            List<OriginEntryFull> originEntries, List<ConcurStandardAccountingExtractDetailLine> detailLines,
+            Consumer<OriginEntryFull> entryConsumer, List<ConcurStandardAccountingExtractDetailLine> detailLines,

```

Comment:
It allows for this class to call CollectorBatch.addOriginEntry(), without passing in the CollectorBatch object itself or creating an intermediate List that the calling code then has to insert. (It may not make sense to pass in the entire CollectorBatch, since this code only needs to invoke that one method on the object and the Consumer allows for that.)